enhancements/accelerators/out-of-tree-drivers.md

```diff
@@ -0,0 +1,418 @@
+---
+title: Supporting out-of-tree drivers on BareMetal & OpenShift
```

Comment:
```suggestion
title: Supporting out-of-tree drivers on OpenShift
```
Bare-metal is a supported platform on OpenShift, and there there is nothing specific here to bare-metal. There is a use-case for kmods in cloud deployments.